HIP 123: cleanup and recommendations (#1048)
helium/HIP: 0123-redefining-data-only-onboarding-and-assertion-fees.md

An off-the-shelf LoRaWAN gateway is capable of transferring traffic through the Helium Network when paired with [gateway-rs](https://github.com/helium/gateway-rs), the light client which enables communication with the Helium Packet Router for PoC-enabled and Data-Only Hotspots alike. This software package can be used without any onboard or assert of the Hotspot. Functionality is illustrated in the table below:

| Gateway State                        | Capabilities                                                               |
|--------------------------------------|----------------------------------------------------------------------------|
| gateway-rs installed and configured. | Data transfer is allowed, but not rewarded to any entity.                  |
| Hotspot is onboarded.                | Data transfer is rewarded to the Hotspot.                                  |
| Hotspot location is asserted.        | Hotspot location is known to the network and returned in payload metadata. |

> Status quo pricing for the onboarding of a Data-Only Hotspot is currently:

Author: ```suggestion
Status quo pricing for the onboarding of a Data-Only Hotspot is:
```

> Current, prior to implementation of this HIP, pricing for the onboarding of a Data-Only Hotspot is currently:

Author: ```suggestion
Status quo pricing for the onboarding of a Data-Only Hotspot is currently:
```

| Action                                            | Cost               |
|---------------------------------------------------|--------------------|
| Data-Only Onboard                                 | 1,000,000 DC ($10) |
| Location Assert (incl elevation and antenna gain) | 500,000 DC ($5)    |
| Transaction Fee                                   | ~0.1 Sol           |

> This HIP acknowledges that data transfer rewards and location metadata might not provide enough incentive for Data-Only Hotspot operators to fully onboard their hardware. Consequently, the Helium Network misses out on valuable data. On-chain knowledge of a Hotspot's location allows it to be included on coverage maps, which the broader Helium community can rely on for their sensor deployments. Furthermore, the precise location of a Hotspot is advantageous for asset tracking applications that rely on multi-lateration.

Author: ```suggestion
This HIP acknowledges that data transfer rewards and location metadata might not provide enough incentive for Data-Only Hotspot operators to fully onboard their hardware. Consequently, the Helium Network misses out on valuable data. On-chain knowledge of a Hotspot's location allows it to be included on coverage maps, which the broader Helium community can rely on for their sensor deployments. Furthermore, the precise location of a Hotspot is advantageous for asset tracking applications that rely on multilateration.
```
